frontend/i18n/en-US.json

```diff
@@ -342,8 +345,12 @@
       "contains-numbers": "Contains numbers",
       "contains-special-chars": "Contains special characters (!#$^*)"
     },
-    "marker-private": {
-      "private": "Private"
+    "view-selector": {
+      "title-aria-label": "Select a view type",
```

Comment:
This is really really wonderful, @bmartins-42 😊 Thanks for the focus on accessibility!